### PR TITLE
Update data-classification-activity-explorer.md

### PR DESCRIPTION
**Pull request by shadansan**
@chrfox and @JohanFreelancer9 I have to make another update to this before publishing. Can you please check and let me know what is the next action? Thank you!

### CODE REVIEW

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/compliance/data-classification-activity-explorer.md [hunk 1]
```diff
@@ -93,7 +93,8 @@ Here's a list of applicable role groups that you can use. To learn more about th
 - Compliance administrator
 - Security administrator
 - Security Reader
-
+- 
+Note that roles other than Global Admin and/or Compliance Admin, can only view activities and are not allowed to view the sensitive content.
```
Comment (chrfox) [inline microsoft-365/compliance/data-classification-activity-explorer.md]: @JohanFreelancer9  - Please put this in Note markdown format  > [!NOTE]

**Comment by chrfox** [comment]
@JohanFreelancer9   please put this in markdown > [!NOTE] format, then send back to me for publishing

File: microsoft-365/compliance/data-classification-activity-explorer.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-activity-explorer.md]:
```suggestion

> [NOTE]
> Roles other than Global Admin and Compliance Admin can only view activities and are not allowed to view sensitive content.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shadansan, Kindly commit the revised suggested changes to incorporate @chrfox's suggestion. Many thanks.

**Comment by shadansan** [comment]
Committed. Please let me know if you need anything else.
Best regards
Shadan

File: microsoft-365/compliance/data-classification-activity-explorer.md [hunk 2]
```diff
@@ -94,6 +94,8 @@ Here's a list of applicable role groups that you can use. To learn more about th
 - Security administrator
 - Security Reader
 
+> [NOTE]
+> Roles other than Global Admin and Compliance Admin can only view activities and are not allowed to view sensitive content.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-activity-explorer.md]:
```suggestion
> Roles other than Global Admin and Compliance Admin can only view activities and are not allowed to view sensitive content.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@shadansan, one more minor change, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@chrfox, I have put the note in markdown format as requested. Could you please check? Many thanks.

**Pull request merged**